### PR TITLE
Update to VS2019

### PR DESCRIPTION
**Pull request by bghgary**

### CODE REVIEW

File: Morphs/Importers/RootPaths.props [hunk 1]
```diff
@@ -14,10 +14,6 @@
     <Import Condition="'$(CanvasRootDirectory)' == ''" Project="CanvasRootDirectory not defined. Define this property before importing $(MSBuildThisFile)." />
   </ImportGroup>
```
Comment (PolygonalSun) [inline Morphs/Importers/RootPaths.props:15]: This PropertyGroup deletion is breaking the Deploy.ps1 script.  From what I've found, updating this with an updated v142 value or modifying the Deploy script to not include the compiler in the lib paths should fix this.

**Review by PolygonalSun — CHANGES_REQUESTED** [review]

File: Morphs/Importers/RootPaths.props (hunk 1 above)
Comment (bghgary) [reply Morphs/Importers/RootPaths.props:15]: I'm not sure I understand why these scripts are here. Who uses them?

Comment (PolygonalSun) [reply Morphs/Importers/RootPaths.props:15]: I believe they were a carryover from when we were building nuget packages.  TBH, we can just cut the script call from the Transcoders post-build.

**Comment by bghgary** [comment]
Defunct since we are using cmake now.

**Pull request closed without merging**